validation/README.md

```diff
@@ -0,0 +1,3 @@
+# Validations
+
+This directory contains standard validations instructions that are reused by multiple tasks.
```

Comment:
NIT: I would maybe add each on in the README as a table.
```suggestion
This directory contains standard validations instructions that are reused by multiple tasks.
- [PRESTATE-VALIDATION.md](./PRESTATE-VALIDATION.md)
```

Reply:
We could add this, but I'm afraid its the kind of thing people will forget to update so it might cause more confusion than it's worth.  

If you feel strongly though, happy to merge your suggestion.

Reply:
Oh I see! nvm I think we can merge like this!